pkg/controller/backup_storage_location_controller.go

```diff
@@ -100,7 +100,8 @@ func (r *backupStorageLocationReconciler) Reconcile(ctx context.Context, req ctr
 	}
 
 	if location.Name == "" || location.Namespace == "" {
-		log.WithError(err).Error("BackupStorageLocation is not found")
+		log.WithError(err).Errorf("BackupStorageLocation %s/%s is not found", req.Namespace, req.Name)
+		log.Infof("Found BSLs: %v", locationList.Items)
```

Comment:
You may want to print better with "%+v" ?

%v (Value):
- Prints the default representation of a value
- For structs, prints just the field values
- Displays maps and slices in a basic format

%+v (Value with field names):
- Prints the default representation with additional detail
- For structs, includes both field names and values
- For errors that implement the fmt.Formatter interface, prints more detailed error information

Reply:
Great point! Made the change.